examples/dynamic/views/Content.js

```diff
@@ -38,6 +44,6 @@ var TodoApp = React.createClass({
       </div>
     );
   }
-});
+}
 
-module.exports = TodoApp;
+module.exports = TodoApp;
```

Comment:
Please leave the trailing newline. You shouldn't have to touch this line at all.